### PR TITLE
Add PersistentVolume in using hive section

### LINKED ISSUE #609: we need to clarify a pv was needed when provision a hive cluster
@clyang82 can you help update the document to reflect this?

### PR DESCRIPTION
**Pull request by clyang82**
Fixed #609
Signed-off-by: Chun Lin Yang <clyang@cn.ibm.com>

### CODE REVIEW

File: docs/using-hive.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
         ```bash
         $ oc create secret generic test-creds --from-literal=aws_secret_access_key=$AWS_SECRET_ACCESS_KEY --from-literal=aws_access_key_id=$AWS_ACCESS_KEY_ID
         ```
+  1. Create a `PersistentVolume` for your ClusterDeployment to store the installation logs. The `accessModes` is `ReadWriteOnce` for your `PersistentVolume`.
```
Comment (gyliu513) [inline docs/using-hive.md]: 1. Can you link `PersistentVolume` to https://kubernetes.io/docs/concepts/storage/persistent-volumes/
2. .s/ClusterDeployment/`ClusterDeployment`

Comment (staebler) [inline docs/using-hive.md]: We should add a note that if the user does not want to capture must-gather logs then they can set `.spec.failedProvisionConfig.skipGatherLogs` to true in the HiveConfig.

**Pull request merged**